.readthedocs.yml

```diff
@@ -0,0 +1,15 @@
+# Required
+version: 2
+
+# Build documentation in the docs/ directory with Sphinx
+sphinx:
+  configuration: docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+formats: all
+
+# Optionally set the version of Python and requirements required to build your docs
+python:
+  version: 3.6
+  install:
+    - requirements: docs/requirements.txt
```

Comment:
We have several libraries that are written in C. To make it easier to build the docs on RTD, I figured I'd limit the number of additional things we install to the smallest number possible. That means there's a tiny bit of duplication between this file and `requirements/*` files.

If we don't update this one, it's no big deal. At worst, the docs won't build, someone will get an email, and we'll fix it. I think that's a better situation than trying to eliminate duplication across requirements files so we can have one set for all things. I predict that'll be harder to do.